.gitignore

```diff
@@ -14,3 +14,6 @@ MATLAB/+qc/personalPaths.mat
 .idea/
 .mypy_cache/*
 tests/hardware/WX2184C.exe
+research_tech/*

```

Comment:
Please do not put personal files here.
`.vscode` is fine but the other two aren't. You can either not commit them or insert these lines into `.git/info/exclude`